README.md

```diff
@@ -21,7 +21,7 @@ When creating Read Replica, make sure to pass the same inputs in the replica ins
 
 ## Usage
 
-See [this example](example/rds.tf)
+See these examples for [MS SQL Server](example/rds-mssql.tf), [MySQL](example/rds-mysql.tf) and [PostGreSQL](example/rds-postgresql.tf).
```

Comment:
```suggestion
See these examples for [MS SQL Server](example/rds-mssql.tf), [MySQL](example/rds-mysql.tf) and [PostgreSQL](example/rds-postgresql.tf).
```

Reply:
Fixed in [0336748](https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance/pull/145/commits/0336748e2cbd3c5cae4701370dc7f961e22cee6b)